sdks/python/apache_beam/runners/common.py

```diff
@@ -95,9 +98,8 @@ def finish(self):
 
   def process(self, element):
     try:
-      with self.logger.PerThreadLoggingContext(step_name=self.step_name):
```

Comment:
I think removing this regresses functionality of the Python SDK relative to Java. We currently ensure that any log message includes the current step in its context to simplify querying, etc.

Is there a plan to re-enable this functionality?